README.md

```diff
@@ -116,39 +117,39 @@ Signed - Academic Director
 
 ## Body Corporate and Contact Details.
 
+| Organisational Contact |                  |
+| :--------------------- | :--------------- |
+| Name                   | Mr George Norris |
+| Position               | Director and CEO |
 
- | Organisational Contact |  |
- | :---- | :---- |
- | Name | Mr George Norris |
- | Position | Director and CEO |
- 
- | Key Academic Contact |    |
- | :---- | :---- |
- | Name | Ms Ruth Green-Cole |
- | Position | Director and Academic Director |
- | Email | ruth@developersinstitute.ac.nz |
- 
- | Campus Address |     |     |
- | :---- | :---- | :---- |
- | Address | 59 Bank Street |    |
- | City | Whangārei | 0110 |
- | Country | New Zealand |    |
- | Phone | (09) 430 0556 |    |
- 
+| Key Academic Contact |                                |
+| :------------------- | :----------------------------- |
+| Name                 | Ms Ruth Green-Cole             |
+| Position             | Director and Academic Director |
+| Email                | ruth@developersinstitute.ac.nz |
+
+| Campus Address |                |      |
+| :------------- | :------------- | :--- |
+| Address        | 59 Bank Street |      |
+| City           | Whangārei      | 0110 |
+| Country        | New Zealand    |      |
+| Phone          | (09) 430 0556  |      |
 
 ## Overview.
+
 ### Ownership Structure and Relationships
 
 Developers Institute Limited is a wholly owned subsidiary of Entelechy Group Limited. All Developers Institute activities and decision-making are separate for Entelechy Group (shareholder) and any other subsidiary of Entelechy Group.
 
 Developers Institute is a registered training establishment by the New Zealand Qualifications Authority under the provisions of the Education Act 1989 and its subsequent amendments.
 
-The Developers Institute is primarily focused on the delivery of web design, web development and software development education within the Northland Area of New Zealand, specifically in Whangārei. 
+The Developers Institute is primarily focused on the delivery of web design, web development and software development education within the Northland Area of New Zealand, specifically in Whangārei.
```

Comment:
@Tegzma I suggest that web design, web development and software development start with capital letters.